packages/volto/src/helpers/Blocks/Blocks.js

```diff
@@ -720,7 +724,7 @@ export const buildStyleObjectFromData = (data = {}, prefix = '') => {
     enhancers.forEach(({ method }) => {
       stylesFromObjectStyleEnhancers = {
         ...stylesFromObjectStyleEnhancers,
-        ...method(data),
+        ...method({ data, container }),
```

Comment:
Isn't this a breaking change that the style enhancer now needs to accept an object with a `data` property instead of a single argument which is `data`?